devicetypes/smartthings/testing/simulated-refrigerator.src/simulated-refrigerator.groovy

```diff
@@ -24,35 +24,61 @@ metadata {
+        childDeviceTile("freezerDoorControl", "freezerDoor", height: 2, width: 2, childTileName: "control")
+		
+        // Row 3: Refrigerator and freezer temperature displays from "Simulated Refrigerator Temperature Control" child
+		childDeviceTile("refrigerator", "refrigerator", height: 2, width: 3) //, childTileName: "refrigerator")
+		childDeviceTile("freezer", "freezer", height: 2, width: 3, childTileName: "freezer")
+        
+        // Row 4: Label for up/down temp controls for refrigerator
+        valueTile("fridgUpDownLabel", "device.contact", width: 1, height: 1, decoration: "flat") {
+            state "contact", label:'Fridge'
+    	}
+        // Row 4: Up/down temp controls for the refrigerator
+        childDeviceTile("refrigeratorUp", "refrigerator", height: 1, width: 1, childTileName: "tempUp")
+		childDeviceTile("refrigeratorDown", "refrigerator", height: 1, width: 1, childTileName: "tempDown")
+        // Row 4: Label for up/down temp controls for the freezer
+        valueTile("freezerUpDown", "device.contact", width: 1, height: 1, decoration: "flat") {
+            state "contact", label:'Freezer'
+    	}
+        // Row 4: Up/down temp controls for the freezer
 		childDeviceTile("freezerUp", "freezer", height: 1, width: 1, childTileName: "tempUp")
 		childDeviceTile("freezerDown", "freezer", height: 1, width: 1, childTileName: "tempDown")
-		childDeviceTile("refrigeratorUp", "refrigerator", height: 1, width: 1, childTileName: "tempUp")
-		childDeviceTile("refrigeratorDown", "refrigerator", height: 1, width: 1, childTileName: "tempDown")
-		childDeviceTile("freezerDoorControl", "freezerDoor", height: 1, width: 1, childTileName: "control")
-		childDeviceTile("mainDoorControl", "mainDoor", height: 1, width: 1, childTileName: "control")
-		childDeviceTile("freezerSetpointUp", "freezer", height: 1, width: 1, childTileName: "setpointUp")
-		childDeviceTile("freezerSetpointDown", "freezer", height: 1, width: 1, childTileName: "setpointDown")
-		childDeviceTile("refrigeratorSetpointUp", "refrigerator", height: 1, width: 1, childTileName: "setpointUp")
+       
+       	// Row 5: "Set Fridge:" label and up/down controls display from "Simulated Refrigerator Temperature Control" child
+		childDeviceTile("refrigeratorSetpoint", "refrigerator", height: 1, width: 1, childTileName: "refrigeratorSetpoint")  
+        childDeviceTile("refrigeratorSetpointUp", "refrigerator", height: 1, width: 1, childTileName: "setpointUp")
 		childDeviceTile("refrigeratorSetpointDown", "refrigerator", height: 1, width: 1, childTileName: "setpointDown")
+       
+        // Row 5: "Set Freezer:" label and up/down controls display from "Simulated Refrigerator Temperature Control" child
+		childDeviceTile("freezerSetpoint", "freezer", height: 1, width: 1, childTileName: "freezerSetpoint")
+        childDeviceTile("freezerSetpointUp", "freezer", height: 1, width: 1, childTileName: "setpointUp")
```

Comment:
This is me being super picky, and maybe others and Bob don't care... but consistent indentation makes a big readability difference. Especially for code that we will be pointing people to as a reference implementation.